lib/compress.js

```diff
@@ -558,7 +559,8 @@ merge(Compressor.prototype, {
 
                     // Only interested in cases with just one reference to the variable.
                     var def = self.find_variable && self.find_variable(var_name);
-                    if (!def || !def.references || def.references.length !== 1 || var_name == "arguments") {
+                    if (!def || !def.references || def.references.length !== 1
+                        || var_name == "arguments" || !toplevel && def.global) {
```

Comment:
Ah, I was wondering who put all those parentheses throughout the codebase 👻

No problem, consider it done 😉